pilot_experiment/3_explore_composition_of_corr_genes.ipynb

```diff
@@ -0,0 +1,1473 @@
5ZcaeCgvpFxnMZWvPUr6ltFkgcWus+xM4MsJ8MtbtHd5DY+UIZDHBqYy5IXideMpF0Lc7/Im6l1VEeUE+OGv9L3VMZ5zncNUrlv3Ib0uxjOa0A7XWXZFxZQT4DtTeGZ9xFaLN7vv8Wa6F7ewUhMayLdNtqeiyqkJ1Z+1sag7zCrXWUzlWFpLz82j2QT8znWW3VFR5QRY+X+0+GArv02HqIhdE+PW2ijp68eTVeFeTWhFrbZRceUEuP9G7X+mkd/kBN+XyDeVoytM5uqJ5LMh7tGEVtwC5hVZToBbbtbOVxu4r2irJ5jt6A+Rv2oiha4Iv9GEdrnOsycqtpwAf/fPuuqNeh4t2h0sZpC8ULx2Atl1MR7w+5maQ6miywlw/a269OUGHi7ayvEGyAmFaybQ/34Nv9OErnKdZ2+IBvOa3912xYVy4PwUp0Yh6jqLcaM/RP6aCWRX1vCQJrTiL/msmnICfPcimXlcitNiStx1FuOvVIjclRPJrolzvyb0I9d5hkJVlRPgkotlv+N7OTOu1LjOYvzRHSb7g4lk2mPcqwld5zrPUKm6cgJc+C2Z+vkezqxR6lxnMd7aFCZzxSQyHVHu0YRudJ1nKFVlOQEWXiKTTuzh9KYyLa6zGG+0R0n/YCKZ7gh3a0KrbknVqi0nwOFXSMs3Ozi9rcAk11nM0Hqlnu6fj6M7F+I+TWhVXoxS1eUEkKREf7qGkw/KMEdAXOcxe6cE5V+PoveBEbwHLNaEVu173FVfzq0uvUjmHZPi+JgSc53F7Jl0iPxP2si8U8fTmtC3XOfx2rApJ8A535ZpC7o5pbFMk+ssZvesidH34/FkOqM8WE1nZHdkWJUTYP7lMuLsTZw6JW/HoZXiuQa6/3Esm7asYBC49WW9MuzKCSBJiVzazrHz+ziyxnZzA6snTPYXY8i+1sDrwEuVdsvX3hqW5dzqxMtk8pmb+WJbgbGus5j/oqDPN9J782i60mEe1YR2us7kwrAuJ4AkJX5ZO8fN6+PTNou61xUmc8NYsm/W8yrwalAXfPbDsC/nVkdfLpPO2swJU3NMdJ1lOCoDTzXRfes+bM4MzJZVd1HB7rJyDiJJiZzdyRGf7eXwMUWaXecZLtbE6LtpNPk/1/ES8MZwni0Hs3JuhySl/sINzDuqj0NaSnZ9rle6wmRu34fs000sB57ThGZcZwoSK+cO7HOttC7s5JjD0hxYV7bb0IZKOkT+vlbSi1pZmw/xlO3Cbp+VcxfMvErGn7WZY+b0s19MibjOU6kyQv6hVvofGEF7NsSzlbyEiB+snLthv6tl2mldHDKnn6nNJWpd56kUm8NkftdC9rEWNvSFeT5oj3cPKivnHqhNyKizN3HooWkOmJS3W9K2R0FX1JC6fwS5Vxp4H1hSqavguWLl3AuSlNoTe5j9mV4OPiDDmAiEXWdyLScUnm8k/WArqY/ivAUs1YTa4mt7wMo5BCQpoRkZpn++l1nTs0yekqMpNIxuTytB+YM4fc81UXyiiXX9YZZoQle7zlXprJxDTJJSMzXL9C/0sN+MLJOm5GgKV8ESpNsqQmlFLennGyi81Mi6rgjvAe9pQnOus1ULK6eHJCk1k3Psf1IP+00fKGpjJZ/tzQnFd2vpf66R7MsNrE2HWQF8YLut3rBy+kSSEq8tM+XoFBMOyNA2Kc+oyTnqajS46+x2hcmsrCH/bi2FZTVsWlbLxrKwClhVzSsQBIWV0xFJSi0w4dN9jP1UhjHj8oweXaRuVJFIY4m4n8esCpoOke+IkH+vhtLyWtJL6+jcGGUdsAZYP9xu1woCK2dASFIEaAZG1JVoPSBL8+QcTaOKNLUUaRpdJNZUIhwrE4rpwJ+oEtlRiUtQLgqlglAuCOVciPLmMKWNUbQzQnFThPTaGH0fxOntjdAFbAI2aEJTfr1u88msnBVCktLIQHnjQGzrn4YSsZYSkcYS0bAS6QtT7g1RTIUpFkIUGXiGzNY/eSAN9GhC045eitlFVk5jAqrqTvEbUy2snMYElJXTmICychoTUFZOYwLKymlMQFk5jQkoK6cxAWXlNCagrJxDTERWiUhGRPpEZIOI/LuINAz6/L+LSFFExm3ne2eJyCIR6RGRlIg8LSLzt/maW0VkhYiURWThUGYQkYNEZLGIdIrIJ146JiL7i0hWRO7cjR+N2U1WTm98SVUbgEOATwPXAohIPXAG0AN8dfA3iMg04EXgHWBfoA14EPiDiMwb9KVvAxcDfxrqDAxcf/tb4Bs72fZNwJKdfI3ZS1ZOD6nqWuD3wEFbPnQG0A38LXDeNl9+HfCyql6jqptVNaWqvwDuAP7voG3epKp/BLJDnUFVV6jqbcCfP2l7IvKVLd//x10Z3+w5K6eHRGQicDLw5pYPnQf8B3APMFNEDh305ScA925nM78FjhKRPVqKczcz7GxbTQyU+rI9yWJ2j5XTGw+JSDfwAvAs8FMRmQQcD9ytqhsYmHnOHfQ9o4D27WyrnYF/pxE+ZNiZ64HbVPXj3cxi9oCV0xunqmqLqk5W1YtVNQN8DVimqm9t+Zq7gLNFZOsyJZ3AfztJtOVjZWB313zdkwyfSETmAJ8HbtjNHGYPVexiUxXoXGCSiGx9BEEEGMnALufDwJPA/wBu3+b7zmLgWHQoHre+sww78hlgCvCRiAA0AGERmaWqhwxBNrMNK6cPtpxtnQbMBToGfeofGCjMw0ASWCIiP9ny8QKwcMvnTxy0rRgDezwCREWkBsir7niNn13JIAOt27rSAlu2raqaA25l4Dh1q8sZKOtFu/hjMLvJyumP84CHVfWdwR8UkRuB50VkhKq+LyJHAz8DVjFQwNeBL6jqi4O+7Q/AcVv+Pp+B0hwPPLO3GYAm4C+DPp0BVgNTtszc/YO+rw/IqurgopshZMuUGBNQdkLImICychoTUFZOYwLKymlMQFk5jQkoK6cxAWXlNCagrJzGBJSV05iA+n/zzifxaM+VmAAAAABJRU5ErkJggg==\n",
```

Comment:
This is a very good point that you raise and one that is rightly very confusing.

Here we are defining core gene = genes that are homologous between PAO1 and PA14. Accessory genes in this case are PAO1-specific genes (since the array we are using is using the PAO1 reference genome).

These shared genes that are accessory genes (PAO1-specific genes) that are highly connected to other accessory genes using only PAO1 samples vs accessory genes (PAO1-specific genes) that are highly connected to other accessory genes only PA14 samples. Hopefully that helps to clarify. 

We are treating the samples separately -- we have our PAO1 samples and PA14 samples. And then we're looking at the correlation of acc-acc within our PAO1 samples and looking at the correlation of acc-acc within our PA14 samples. And looking to see if the genes with high acc-acc correlation are the same in both datasets.

Reply:
![Mtg_2020_06_05 (1)](https://user-images.githubusercontent.com/13107360/84806905-6b849500-afd4-11ea-9f90-3e07ed0c7c71.jpg)

Reply:
Ok, that's helpful. So then I guess my question is: what is an accessory gene that is "shared" between PAO1 and PA14? Naively I would think shared == homologous, but if a gene was homologous between strains it would be a core gene, wouldn't it?

If not, how are you defining "shared" here?

Reply:
Based on discussion I have updated the text to clarify what these "shared genes" are. Thank you for the discussion!